app/api/v1/mediator/ballot.py

```diff
@@ -43,6 +48,40 @@ def spoil_ballot(request: AcceptBallotRequest = Body(...)) -> Any:
     return spoiled_ballot.to_json_object()
 
 
+@router.post("/encrypt", tags=[ENCRYPT_BALLOTS])
+def encrypt_ballots(request: EncryptBallotsRequest = Body(...)) -> Any:
+    """
+    Encrypt one or more ballots
+    """
+    ballots = [PlaintextBallot.from_json_object(ballot) for ballot in request.ballots]
+    description = InternalElectionDescription(
+        ElectionDescription.from_json_object(request.description)
+    )
+    context = CiphertextElectionContext.from_json_object(request.context)
+    seed_hash = read_json_object(request.seed_hash, ElementModQ)
+    nonce: Optional[ElementModQ] = (
+        read_json_object(request.nonce, ElementModQ) if request.nonce else None
+    )
+
+    encrypted_ballots = []
+    current_hash = seed_hash
+
+    for ballot in ballots:
+        encrypted_ballot = encrypt_ballot(
+            ballot, description, context, current_hash, nonce
+        )
+        if not encrypted_ballot:
+            raise HTTPException(status_code=500, detail="Ballot failed to encrypt")
+        encrypted_ballots.append(encrypted_ballot)
+        current_hash = get_optional(encrypted_ballot.tracking_hash)
+
+    response = EncryptBallotsResponse(
+        encrypted_ballots=[ballot.to_json_object() for ballot in encrypted_ballots],
+        tracking_hash=write_json_object(current_hash),
```

Comment:
I like adding the hash here, but I feel like the name could be confusing

Why not leave it as `current_tracking_hash` or `previous_tracking_hash` since it's the last hash from the set?